docker-host/Dockerfile

```diff
@@ -23,17 +35,12 @@ RUN apk add --no-cache bash
 
 ENV DOCKER_HOST=docker-host \
     REGISTRY=docker-registry.default.svc:5000 \
-    REPOSITORY_TO_UPDATE=amazeeio \
+    REPOSITORIES_TO_UPDATE=amazeeio \
```

Comment:
with the new filter, this should update to match what the new filters are expecting

Reply:
(and be uselagoon by default :wink:)